components/prism-ignore.js

```diff
@@ -0,0 +1,34 @@
+(function (Prism) {
+	Prism.languages.ignore = {
+		// https://git-scm.com/docs/gitignore
+		'header': {
+			pattern: /(^|[^\\])###.*/,
+			lookbehind: true,
+			alias: ['comment', 'bold']
+		},
+		'section': {
+			pattern: /(^|[^\\])##.*/,
+			lookbehind: true,
+			alias: 'comment'
+		},
+		'comment': {
+			pattern: /(^|[^\\])#.*/,
+			lookbehind: true,
+			alias: 'namespace'
```

Comment:
What? Please let the themes style comments as comments.

Reply:
It's just a hack to make comments semi-transparent.
I wait for your decision about sections and headers and then I'll fix it

Reply:
Removed alias